app/directives/selectionList/selectionList.directive.html

```diff
@@ -1,11 +1,17 @@
-<div class="db-directive">
-    <section class="db-margin-15" ng-repeat="data in c.data track by $index">
-        <header ng-if="data.title.header" translate translate-cloak>{{data.title.header}}</header>
-        <ul role="menu">
-            <li role="menuitem" class="ng-class:[{'db-opacity-05':c.isDisabled(att)}]" ng-repeat="att in data.objects track by $index">
-                <a class="" ng-click="!c.isDisabled(att) && c.selected(att)"><span ng-if="att.orderNum">{{att.orderNum}}&nbsp;</span>{{att.title}}</a>
+<div class="db-directive" ng-init="isCollapsed = false">
+
+    <ul class="list-group db-margin-10-bottom" ng-repeat="data in c.data track by $index" ng-click="isCollapsed = !isCollapsed">
```

Comment:
ngClick probably shouldn't be on the `ul` level and instead on the next header `li` level so that clicking the attachment rows won't collapse the list